source/extensions/filters/network/thrift_proxy/filters/filter.h

```diff
@@ -142,12 +142,44 @@ class EncoderFilterCallbacks : public virtual FilterCallbacks {
   virtual void continueEncoding() PURE;
 };
 
+/**
+ * Return codes for onLocalReply filter invocations.
+ */
+enum class LocalErrorStatus {
+  // Continue sending the local reply after onLocalError has been sent to all filters.
+  Continue,
+
+  // Continue sending onLocalReply to all filters, but reset the stream once all filters have been
+  // informed rather than sending the local reply.
+  ContinueAndResetStream,
```

Comment:
Is the description correct? Seems like `parent_.sendLocalReply` is always called and this causes the stream to be reset after the reply.

Reply:
you're right. Let me see which makes sense, changing description or implementation.